src/internal/kopia/merge_details.go

```diff
@@ -44,6 +44,7 @@ func (m *mergeDetails) ItemsToMerge() int {
 
 func (m *mergeDetails) addRepoRef(
 	oldRef *path.Builder,
+	modTime *time.Time,
```

Comment:
pointer for mutation, or to denote missing/absent data?  Could you add a comment to explain which is expected?